staging/src/k8s.io/apiserver/pkg/authentication/token/cache/cached_token_authenticator_test.go

```diff
@@ -249,9 +250,7 @@ func (s *singleBenchmark) makeTokens() {
 }
 
 func (s *singleBenchmark) lookup(ctx context.Context, token string) (*authenticator.Response, bool, error) {
-	s.chokepoint <- struct{}{}
```

Comment:
I don't particularly mind this change, but it seems best to leave the benchmark constant while you're making code changes, to be 100% sure that the numbers reflect the code and not the benchmark change?

Reply:
Moved to a seperate PR.